tests/gold_tests/pluginTest/lua/lua_states_stats.test.py

```diff
@@ -23,16 +23,25 @@
 Test.SkipUnless(
     Condition.PluginExists('tslua.so'),
 )
-Test.SkipIf(Condition.true("Test cannot deterministically wait until the stats appear"))
 
 Test.ContinueOnFail = True
 # Define default ATS
 server = Test.MakeOriginServer("server")
 
-ts = Test.MakeATSProcess("ts", command="traffic_manager", select_ports=True)
+# It is necessary to redirect stderr to a file so it will be available for examination by a test run.
+ts = Test.MakeATSProcess(
+    "ts", command="traffic_manager 2> " + Test.RunDirectory + "/ts.stderr.txt", select_ports=True
```

Comment:
We already have this text being grabbed in the Streams object. We should have examples test using later for a given test

Reply:
The lifecycle_stats.sh script needs to read the stderr output from traffic_manager.  I tried reading it from the stderr file in _output for the ts process, but the output the script needs gets buffered in memory.  In any case, not good to hardcode the sandbox tree structure into tests.